site/components/JustifiedBetweenRow.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const JustifiedBetweenRow = ({ keyComponent, valueComponent }) => (
+  <div className="flex justify-between">
+    {keyComponent}
+    {valueComponent}
+  </div>
+)
+
+export default JustifiedBetweenRow
```

Comment:
I know I've seen this component elsewhere, but IMO it does not add too much value (only two tailwind clases)... I think it's better to apply it inline whenever needed (plus, if I need a Flex that is not justified between we'll have to inline it or create another component).

What do you think?

Reply:
Agree. @mbonfoster?

Reply:
Yeah, agree. It's not something that bothers me a lot. I'm ok removing it and also ok keeping it.

Reply:
Ok. Let's remove it from here and from the other apps in a different PR so we close this ASAP.